layers/meta-tegrademo/scripts/setup-udev-rules

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+user=$(who am i | awk '{print $1}')
+if (( $EUID != 0 )); then
+    echo "You may need to run this root as sudo if you see permission errors"
+fi
+groups ${user} | grep "plugdev" > /dev/null 2>&1
+set -e
+if [ $? -ne 0 ]; then
+    echo "Adding ${user} to plugdev"
+    usermod -a -G plugdev  ${user}
+fi
```

Comment:
This seems a little dangerous to me, to do this automatically... maybe a prompt first? They'll need a reminder to log out and back in again for the added group to take effect, anyway.

Reply:
Good point, I've replaced with prints that tell you what to do instead.